src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -695,7 +713,19 @@ public DescriptorImpl() {
             if (repoJobs == null) {
                 repoJobs = new ConcurrentHashMap<String, Set<AbstractProject<?, ?>>>();
             }
-            save();
+            saveAfterPause();
+        }
+        
+        private void saveAfterPause() {
```

Comment:
Technically in development the version is really old, and I am not sure if it is even supported anymore.